ethereum/contracts/EtherVault.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+// Holds ETH on behalf of ETHApp
+contract EtherVault is Ownable {
+    event Deposit(address account, address origin, uint256 amount);
```

Comment:
Similar to the [Message event](https://github.com/Snowfork/snowbridge/pull/699#discussion_r995592247) in the outbound channel contract, should we avoid the name `origin` to avoid the `tx.origin` association? We could use `sender` to match Withdraw's `recipient`
cc @vgeddes